hookman/hookman_generator.py

```diff
@@ -284,6 +285,12 @@ def _validate_plugin_config_file(cls, plugin_config_file: Path, artifacts_dir: P
         """
         plugin_file_content = PluginInfo(plugin_config_file, hooks_available=None)
 
+        semantic_version_re = re.compile(r'^(\d+)\.(\d+)\.(\d+)')  # Ex.: 1.0.0
+        version = semantic_version_re.match(plugin_file_content.version)
+
+        if not version:
+            raise ValueError(f"Version attribute does not follow semantic version, got {plugin_file_content.version}")
```

Comment:
I suggest to use quotes here so the version stands out:

```suggestion
            raise ValueError(f"Version attribute does not follow semantic version, got {plugin_file_content.version!r}")
```